setup.py

```diff
@@ -9,7 +9,7 @@
     python_requires='>=3.6',
     # via environment variables, in which case I've tested with 15.0.4
     # "x11_hash" package (1.4) is required to sync DASH network.
-    install_requires=['plyvel', 'pylru', 'aiohttp >= 1'],
+    install_requires=['plyvel', 'pylru', 'tribus_hash', 'aiohttp >= 1'],
```

Comment:
Please leave this out.  Users will have to install this manually if they want it.  Note there is no x11_hash, or scrypt specified there either.